src/main/java/com/bobocode/orm/action/DeleteAction.java

```diff
@@ -0,0 +1,24 @@
+package com.bobocode.orm.action;
+
+import com.bobocode.orm.jdbc.JDBCRepository;
+
+public class DeleteAction implements EntityAction {
+
+  private final Object entity;
+  private final JDBCRepository jdbcRepository;
+
+  public DeleteAction(Object entity, JDBCRepository jdbcRepository) {
```

Comment:
MINOR. In case you're using Lombok you might use @RequiredArgsConstructor

Reply:
addressed